packages/core/cache/nuxt/index.js

```diff
@@ -28,15 +28,18 @@ function createInvalidationEndpoint (driver, options) {
         tags: Array.from(new Set(tags))
       });
 
-      response.writeHead(200);
+      response.status(200).send('Cache invalidated successfully!');
```

Comment:
```suggestion
      response
        .status(200)
        .send('Cache invalidated successfully!');
```

Just to be consistent

Reply:
Corrected in [41c9831](https://github.com/vuestorefront/vue-storefront/pull/6353/commits/41c9831c9ec2be830f7db5fbd44acd84c959f8b8)